scripts/get_books.py

```diff
@@ -0,0 +1,55 @@
+""" Script to generate a json file containing book names  """
+
+import json
+import urllib.request
+import re
+
+RESULT_JSON_NAME = "books.json"
+
+# Get catalog.json
+URL_CAT = "https://api.unfoldingword.org/uw/txt/2/catalog.json"
+response_cat = urllib.request.urlopen(URL_CAT)
+DATA = json.loads(response_cat.read().decode('utf-8'))
+
+OUTPUT = []
+
+LANGS = DATA["cat"][0]["langs"]
+
+# just a pointer, since the full path is dynamic-ish
+langIdx = []
+for idx in range(len(LANGS)):
+    if LANGS[idx]['lc'] == 'en':
+        langIdx = LANGS[idx]
+
+# skip obs for now, loop over all books
+for x in range(0, 66):
+    # gives book name and order (the books are stored out of order in the json)
+
+    # Having to navigate through DATA a bit because the location of English may move as more languages are added
+    # Using English because the list of books may not be complete in other languages
+    # 0 index after cat is for Bible, 1 contains data for obs
```

Comment:
i don't see `cat` do we still need this comment here?

Reply:
@natibekele I have no idea, these scripts were written by Craig